entity-framework/core/extensions/index.md

```diff
@@ -54,6 +54,14 @@ Entity Framework Core Generator (efg) is a .NET Core CLI tool that can generate
 
 ## Extensions
 
+### Microsoft.EntityFrameworkCore.DistributedCache
```

Comment:
This isn't feedback on the PR, but on the name of the package:

- It shouldn't really start with "Microsoft." or contain "Microsoft"in the name unless the package is from Microsoft. For example, we recommend for example, provider writer, to use "Company.EntityFrameworkCore.Database"  for the provider packages. I know there are some in this page that don't follow this rule. That is because it wasn't enforced before. 
- I noticed that in the actual package you use a prefix "Extention". Is that the name of the organization that creates it or a just "Extension" misspelled?